agent/tests/unit/test_task_routing_inheritance.py

```
@@ -0,0 +1,90 @@
+import sys
+import types
+import unittest
+from datetime import datetime, timezone, timedelta
+
+sys.modules.setdefault("aiohttp", types.SimpleNamespace(ClientSession=None))
+
+from services.task_service import TaskService
```

Ruff (0.15.12)

[error] Module level import not at top of file

(E402)